### PR TITLE
[wip] Flytekit only plugin

### PR DESCRIPTION
**Pull request by wild-endeavor**

### CODE REVIEW

File: rsts/howto/template_only_tasks.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
+.. _howto-template-only-tasks:
```
Comment (kumare3) [inline rsts/howto/template_only_tasks.rst:1]: This does not belong to how to - but rather in - https://docs.flyte.org/projects/cookbook/en/latest/extending_flyte.html

Comment (kumare3) [reply rsts/howto/template_only_tasks.rst:1]: we should also update the decision flowchart to add this as a new type of extension possibility

File: rsts/howto/template_only_tasks.rst [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+Process Differences
+=====================
+
+Normal function tasks
+---------------------
+
+Most tasks that are in the cookbook and other Flyte introductory material are basic Python function tasks. That is, they are created by decorating a Python function with the ``@task`` decorator. Please see the basic Task concept doc for more details. With the decorator in place, the process is
+
+#. At serialization time, a Docker container image is required. The assumption is that this Docker image has the task code.
+#. The task is serialized into a ``TaskTemplate``. This template contains instructions to the container on how to reconstitute the task.
```
Comment (kumare3) [inline rsts/howto/template_only_tasks.rst:25]:
```suggestion
#. The task is serialized into a :std:ref:`api_msg_flyteidl.core.tasktemplate`. This template contains instructions to the container on how to reconstitute the task.
```

File: rsts/howto/template_only_tasks.rst [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+Background
+**********************
+
+Process Differences
+=====================
+
+Normal function tasks
+---------------------
+
+Most tasks that are in the cookbook and other Flyte introductory material are basic Python function tasks. That is, they are created by decorating a Python function with the ``@task`` decorator. Please see the basic Task concept doc for more details. With the decorator in place, the process is
```
Comment (kumare3) [inline rsts/howto/template_only_tasks.rst:22]: replace @task with the reference to flytekit?

Comment (kumare3) [reply rsts/howto/template_only_tasks.rst:22]: :py:function:`flytekit.task`

File: rsts/howto/template_only_tasks.rst [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+Image
+=======
+This is the custom image that you supplied in the ``PythonCustomizedContainerTask`` subclass. Out of the box, these tasks will run a command that looks like the following when the container is run by Flyte ::
+
+    pyflyte-execute --inputs s3://inputs.pb --output-prefix s3://outputs --raw-output-data-prefix s3://user-data --resolver flytekit.core.python_customized_container_task.default_task_template_resolver -- {{.taskTemplatePath}} path.to.your.executor.subclass
+
+This means that your Docker image will need to have Python and Flytekit installed. The Python interpreter that is run by the container should be able to find your custom executor class at that ``path.to.your.executor.subclass`` import path.
+
+Feel free to take a look at the Flytekit Dockerfile as well.
```
Comment (SandraGH5) [inline rsts/howto/template_only_tasks.rst:107]: Is it possible to include a link to the Docker file?

Comment (kumare3) [reply rsts/howto/template_only_tasks.rst:107]: yes it is. I dont know why we are copying the dockerfile to readme
A better way is to use the literalinclude directive - https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude

Comment (kumare3) [reply rsts/howto/template_only_tasks.rst:107]: or the remote include

**Comment by kumare3** [comment]
@wild-endeavor is this ready to be reviewed?

**Comment by samhita-alla** [comment]
closing this PR as the content has been covered in https://github.com/flyteorg/flytesnacks/pull/744.

**Pull request closed without merging**